homework_protos/homework_service.proto

```diff
@@ -0,0 +1,72 @@
+syntax = "proto3";
+
+option go_package ="github.com/BetterGR/homework-microservice/homework_protos";
+
+package Homework;
+/*
+The Homework domain manages all information related to assignments for each course,
+including workflows for completing homework, submission tracking
+ */
+service HomeworkService{
+    //returns all the homework assigned in a certain course
+    rpc GetHomework(GetHomeworkRequest) returns (GetHomeworkResponse);
+    //creates a new homework for a certain course and returns boolean response
+    rpc CreateHomework(CreateHomeworkRequest) returns (CreateHomeworkResponse);
+}
+
+
+// Definition of Homework message
+message Homework {
+    string id = 1;                // Unique ID for the homework

```

Comment:
Field descriptive comments should be before the field.

E.g.,:
```go
// id is a unique identifier for the homework.
string id = 1;
```

Note that the comment is dotted as well.